delegates.txt

```diff
@@ -19,6 +19,7 @@ Andrew Burgese (ABE)
 Andrew Burgess (ABS)
 Andrew Paprocki (API)
 Andy Fleming (AFG)
+Angela Wang(AW)
```

Comment:
```suggestion
Angela Wang(AWG)
```

We are no longer adding new 2 letter delegate abbreviation. This is the format we have generally followed for 3 letter (First letter of first name, First letter of last name, last letter of last name)